word_guess.rb

```diff
@@ -0,0 +1,143 @@
+
+require 'colorize'
+
+class NewGame
+	# method to start new game with randomly selected word
+	def initialize(word)
+		@word = word
+    	@game_status = true
+    	@letter_array = []
+    	@dashes_array = []
+		@incorrect_guesses = 0
+		@already_guessed = []
+		@icecream = ["     ()".colorize(:magenta), "    (__)".colorize(:yellow), "   (____)".colorize(:green), "  (______)".colorize(:cyan), " (________)".colorize(:light_blue), "(__________)".colorize(:red), " X X X X X".colorize(:yellow), "  X X X X ".colorize(:yellow), "   X X X ".colorize(:yellow), "    X X ".colorize(:yellow), "     X".colorize(:yellow)]
+		user_guess
+	end
+
+	def user_guess
+		while @game_status == true do
+			draw_icecream
+        	draw_guesses
+			puts "\nGuess a letter:"
+			guess = gets.chomp.downcase
+			# make sure user enters letter a-z
+			until guess.match(/[a-zA-Z]/)
+				puts "Please guess a letter from a-z:"
+				guess = gets.chomp.downcase
+			end
+			# if already guess letter, reprompt and don't lose round
+			while @already_guessed.include? guess
+				puts "You already guessed that letter, try another!".colorize(:cyan)
+				puts "\nGuess a letter:".colorize(:cyan)
+				guess = gets.chomp
+			end
+			@already_guessed.push(guess)
+			guess_include(guess)
+
+		end
+	end
+
+	def guess_include(guess)
+
+		# if one or more occurances of guess is in word or guess is whole word
+		if @letter_array.include?(guess) || guess == @word
+			#puts "It's a match!"
+
+			# if the guess is the whole word, win and exit
+			if guess == @word
```

Comment:
It seems like this `if` statement doesn't need to be nested inside of the first one. It seems to be a different case than if the user has guessed a single letter, so i think this can come out to the top (not nested inside another `if`).